doc/book/connectors/__python.rst

```diff
@@ -44,3 +44,214 @@ Also there are several community-driven Python connectors:
+
+Feature comparison
+------------------
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
+
+    *   -   Parameter
+        -   `igorcoding/asynctnt <https://github.com/igorcoding/asynctnt>`__
+        -   `shveenkov/aiotarantool <https://github.com/shveenkov/aiotarantool>`__
+        -   `shveenkov/gtarantool <https://github.com/shveenkov/gtarantool>`__
+        -   `tarantool/tarantool-python <https://github.com/tarantool/tarantool-python>`__
+
+    *   -   License
+        -   Apache License 2.0
+        -   LGPL
+        -   LGPL
+        -   Modified BSD
+
+    *   -   Is maintained
+        -   Yes
+        -   No (last updated in 2016)
+        -   No (last updated in 2018)
+        -   Yes
+
+    *   -   Estimation of importance of known bugs
+        -   `issue #18 <https://github.com/igorcoding/asynctnt/issues/18>`__
+        -   `issue #5 <https://github.com/shveenkov/aiotarantool/issues/5>`__
+            (select() fails), `issue #16
+            <https://github.com/shveenkov/aiotarantool/issues/16>`__ (Tarantool
+            1.7 is unsupported)
+        -   No known bugs
+        -   `issue #105 <https://github.com/tarantool/tarantool-python/issues/105>`__ (Unpack of binary data)
+
+    *   -   Documentation
+        -   Yes (`github.io <https://igorcoding.github.io/asynctnt/>`__)
+        -   No
+        -   No
+        -   Yes (`tarantool.io
+            <https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/>`__
+            and `readthedocs
+            <https://tarantool-python.readthedocs.io/en/latest/quick-start.en.html>`__
+            (`obsolete
+            <https://github.com/tarantool/tarantool-python/issues/67>`__))
+
+    *   -   Testing / CI / CD
+        -   Github Actions
+        -   No (tests exists)
+        -   No (tests exists)
+        -   Travis CI and Appveyor
+
+    *   -   Github Stars
+        -   51
+        -   38
+        -   17
+        -   78
+
+    *   -   Static Analysis
+        -   Yes (Flake8)
+        -   No
+        -   No
+        -   No
+
+    *   -   Packaging
+        -   `Pip <https://pypi.org/project/asynctnt/>`__
+        -   `Pip <https://pypi.org/project/aiotarantool/>`__
+        -   `Pip <https://pypi.org/project/gtarantool/>`__
+        -   `Deb, RPM, Pip <https://github.com/tarantool/tarantool-python#download-and-install>`__
+
+    *   -   Code coverage
+        -   Yes (no visualization)
+        -   No
+        -   No
+        -   Yes (no visualization)
+
+    *   -   Support asynchronous mode
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes? (`gevent
+            <https://www.gevent.org/api/gevent.event.html#gevent.event.AsyncResult>`__,
+            example: `test_gevent.py
+            <https://github.com/shveenkov/gtarantool/blob/master/tests/test_gevent.py>`__)
```

Comment:
I think we can say 'Yes, gevent' without the question mark.

Reply:
removed question mark